server/command.go

```diff
@@ -16,6 +16,7 @@ const helpText = "###### Mattermost Jira Plugin - Slash Command Help\n" +
 	"\nFor system administrators:\n" +
 	"* `/jira install cloud <URL>` - connect Mattermost to a cloud Jira instance located at <URL>\n" +
 	"* `/jira install server <URL>` - connect Mattermost to a server Jira instance located at <URL>\n" +
+	"* `/jira assign <issuekey> <assignee>` - Changes the assignee of a Jira issue.\n" +
```

Comment:
Is it restricted to system administrators? If so, you need to check in executeAssign.

Reply:
Good question.   @jasonblais?  I think it should be left open.  The slash command will check if the user has jira permissions on the issue.

Reply:
No need to restrict to system administrators. It should be dependent on Jira permissions, similar to `/jira transition`

Reply:
Thanks @jasonblais.  Will leave as is.  Thanks for the question too, @cpoile